pwa/src/templates/components/ComponentsTemplate.tsx

```diff
@@ -88,9 +103,16 @@ export const ComponentsTemplate: React.FC = () => {
         </FormField>
       </form>
 
-      {filteredComponents.length > 0 && <ComponentResultTemplate results={filteredComponents} type={display} />}
+      {getComponents.isSuccess && (
+        <>
+          {filteredComponents.length > 0 && (
```

Comment:
`filteredComponents.length` can only be `> 0` if `getComponents.isSuccess`. Therefore, we've created a redundant check here.

A possible solution would be:

```
{filteredComponents.length > 0 && (
        <ComponentResultTemplate components={filteredComponents} type={filters.resultDisplayLayout} />
      )}

      {!filteredComponents.length && !getComponents.isLoading && t("No components found with active filters")}
      {getComponents.isLoading && <Skeleton height="200px" />}
```